doc/rsp.md

```diff
@@ -280,95 +315,168 @@ mapped into the 16-bit lanes. Signed opcodes (`LPV`, `SPV`) map the value to bit
+| `LFV` | 0x09 | load 8 unsigned 8-bit values into 8 lanes |
+| `SHV` | 0x08 | store 8 unsigned 8-bit values from 8 lanes |
+| `SFV` | 0x09 | store 8 unsigned 8-bit values from 8 lanes |
+
+Similar to (`LUV`, `SUV`), these handle unsigned values, and map each value to bits
+`(14..7)` of the 16-bit lanes. Load instructions zero the bits of each lane outside
+the mapped range, while store instructions effectively ignore the other bits.
+
+Like the 8-bit packed loads, the strided loads can be viewed as 128-bit loads from
+`GPR[base] + (offset * 16)` that are then rotated based on alignment and element.
+After loading and rotating as above, however, instead of mapping the leftmost 8 bytes
+into the lanes, every other byte, in the case of `LHV`, and every fourth byte
+(repeated in a pattern), in the case of `LFV`, are used.
+
+`LFV`, since it doesn't write an entire register, behaves differently from other loads
+at this last step. It first creates a new 128-bit temporary, and loads `(14..7)` of each
+lane in the temporary from a different byte in the rotated value, using the pattern
+`0,4,8,12,8,12,0,4`. 64 bits of the original register starting at byte index `element`
+(NOT wrapping around) are then replaced with the corresponding bits of this temporary.
+
+`SHV` stores the appropriate bits of each lane into every other byte in memory, like
+`SUV`. However, instead of storing one lane at a time starting from lane `element`,
+it stores every other byte in the register, beginning at byte index `element`
+(after rotating the entire register left by one bit to align the mapping). Mapping
+bits are not affected by `element`, and addresses wrap at the second 64-bit boundary.
+
+`SFV` is more complex. It first creates a new 128-bit temporary, and loads each byte
+of the temporary from `(14..7)` of a different lane in the source register, using the
+pattern `0,6,X,X,1,7,X,X,2,4,X,X,3,5,X,X` (`X` repesents a zeroed byte). It then
+increments `element` values in the range `[8..15]` by 1, so they become `[9..15,0]`.
+Finally, every fourth byte of the temporary, beginning at byte index `element`,
+is written to every fourth byte in memory, with addresses wrapping at the second 64-bit
+boundary.
+
+For instance:
+
+    // a0 is 64-bit aligned
+    LHV v1[e7],$06(a0)     // load bytes $01(a0)..$0d(a0) (odd) into VPR[1]<1..7>,
+                              then byte $0f(a0) into VPR[1]<0>
+    SHV v1[e3],$06(a0)     // write bytes $06(a0)..$0e(a0) (even) from VPR[1][3..11] (odd)
+                              then bytes $00(a0)..$04(a0) (even) from VPR[1][13,15,1]
+    LFV v1[e3],$06(a0)     // load byte $07(a0) into VPR[1]<1>
+                              then bytes $0b(a0),$0f(a0) into VPR[1]<2..3>
+                              then bytes $0b(a0),$0f(a0) into VPR[1]<4..5>
+    SFV v1[e5],$06(a0)     // write bytes $06(a0),$0a(a0),$0e(a0) from VPR[1]<7,4,5>
+                              then byte $02(a0) from VPR[1]<6>
 
```

Comment:
I think this can go in as-is. I still don't specifically like the "rotation" approach to description, but since there was nothing before, I can't really complain, this is surely a good description anyway.